src/state_machine.zig

```diff
@@ -1729,18 +1637,19 @@ test "create/lookup/rollback transfers" {
     defer testing.allocator.free(output);
 
     _ = state_machine.prepare(.create_accounts, input);
-    const size = state_machine.commit(0, 0, .create_accounts, input, output);
+    //const size = state_machine.commit(0, 0, .create_accounts, input, output);
 
-    const errors = mem.bytesAsSlice(CreateAccountsResult, output[0..size]);
-    try expect(errors.len == 0);
+    //const errors = mem.bytesAsSlice(CreateAccountsResult, output[0..size]);
+    //try expect(errors.len == 0);
 
     for (accounts) |account| {
+        state_machine.forest.grooves.accounts.put(&account);
         try expectEqual(account, state_machine.get_account(account.id).?.*);
     }
 
     const Vector = struct { result: CreateTransferResult, object: Transfer };
 
-    const timestamp: u64 = (state_machine.commit_timestamp + 1);
+    const timestamp: u64 = state_machine.prepare_timestamp + 1;
```

Comment:
Should this not be `commit_timestamp`?

Reply:
In order to simplify the setup for the transfers test, I `put` into the accounts groove directly. But that means the `commit_timestamp` isn't updated — the `prepare_timestamp` is, though.